### ISSUE TITLE
Feature: Provide Cross section library for running tests

### ISSUE THREAD

**Issue by yardasol** [arfc/saltproc#76]
**Background and motivation**
See #72 for background.
The TL;DR is that we currently do not specify in the docs any particular cross section library that is necessary for the tests, even though the tests themselves are set up to pass for a particular library. We should do two things:
1. Specify in the docs that users need to download and set up a specific library in order for the tests to pass
2. Provide the library to the users if it requires more work than downloading, extracting, and modifying `.bashrc`.

**Description of idea**
In #72 I detailed my process for reconstructing the JEFF 3.1.2 data into a library that can be used with Serpent. I suggest we use this library and put it somewhere on the saltproc website, or link to it straight from the repo.

**Implementation details**
We need to add a section in the docs somewhere that talks about how to set up the library, and specify that it is required for the test suite to pass.

We also need to host a downloadable archive of the library somewhere. @munkm any ideas?
 
**Potential snags**
I'm still working through getting the integration tests up and running again, so the library isn't actually ready yet. Once #72 is closed, we can work on making a PR to resolve _this_ issue.

**Comment by yardasol** [comment]
Closing this issue as we aren't going to be hosting our own cross section library after all. See #78 and #79.
